jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -228,61 +240,81 @@ def check_cudnn_version(is_flash_attention, is_cross_attention):
   elif not is_flash_attention and cuda_versions.cudnn_get_version() < 8901:
     raise RuntimeError("JAX requires cuDNN >= 8.9.1 to use fused attention.")
 
-def _dot_product_attention_fwd(query, key, value, bias, mask,
-  scale, seed, dropout_rate, variadic_args, is_flash_attention, is_causal_mask):
+def _dot_product_attention_fwd(
+    query, key, value, bias, mask, scale, seed, dropout_rate, variadic_args,
+    is_flash_attention, is_causal_mask, layout):
   output, _ = _dot_product_attention_fwd_p_wrapper.bind(
-    query, key, value, bias, mask, scale=scale, seed=seed, dropout_rate=dropout_rate,
-    variadic_args=variadic_args, is_flash_attention=is_flash_attention,
-    is_causal_mask=is_causal_mask)
+      query, key, value, bias, mask, scale=scale, seed=seed,
```

Comment:
for format changes like this, maybe include in a separate pr so reviewers can review the changes related to layout changes easily.

Reply:
When I add the `layout=layout` to the callsite, the pylint will give format violation warning. Do you mean we should ignore all format issues? I thought for every commit we should follow the google code style, like https://google.github.io/styleguide/pyguide.html.